server/mldb_server.h

```diff
@@ -142,6 +143,28 @@ struct MldbServer: public ServicePeer, public EventRecorder {
     Utf8String prefixUrl(Utf8String url) const;
     std::string prefixUrl(std::string url) const;
     std::string prefixUrl(const char* url) const;
+    
+    InProcessRestConnection mldbPerform(
```

Comment:
Why do we use the mldb prefix here?  It doesn't make sense: we know it's MLDB because this is an MldbServer.